src/groundlight/client.py

```diff
@@ -172,11 +173,62 @@ def list_image_queries(self, page: int = 1, page_size: int = 10) -> PaginatedIma
             image_queries.results = [self._fixup_image_query(iq) for iq in image_queries.results]
         return image_queries
 
+    def ask_confident(
+        self,
+        detector: Union[Detector, str],
+        image: Union[str, bytes, Image.Image, BytesIO, BufferedReader, np.ndarray],
+        wait: Optional[float] = None,
+    ) -> ImageQuery:
```

Comment:
Intuitively I feel like I should be able to specify the confidence threshold here.  I know it's set on the detector.  But IIRC the API lets us override it for a specific IQ.  And given the method signature it feels like I should be able to include it.